Content.Client/Shipyard/BUI/ShipyardConsoleBoundUserInterface.cs

```diff
@@ -0,0 +1,91 @@
+using Content.Client.Shipyard.UI;
+using Content.Client.Shipyard.Components;
+using Content.Shared.Shipyard.BUI;
+using Content.Shared.Shipyard.Events;
+using Robust.Client.GameObjects;
+using Robust.Client.Player;
+using Robust.Shared.Utility;
+using Robust.Shared.Prototypes;
+using static Content.Shared.Shipyard.Components.SharedShipyardConsoleComponent;
+using static Robust.Client.UserInterface.Controls.BaseButton;
+
+namespace Content.Client.Shipyard.BUI
+{
+    public sealed class ShipyardConsoleBoundUserInterface : BoundUserInterface
+    {
+        [Dependency] private readonly IEntityManager _entityManager = default!;
+
+        [ViewVariables]
+        private ShipyardConsoleMenu? _menu;
+
+        [ViewVariables]
+        public int Balance { get; private set; }
```

Comment:
I have no idea, I just see it added to most BUIs so I figured it was necessary?

Reply:
I had a look at the cargo ordering console, since it does use ViewVariables on its properties. I've found out you can look at the BUI's properties in game through VV if you access the ClientUserInterface component and go into the interfaces available. After seeing that, I wouldn't say it's necessary, but you can surely put it in if you want to debug these properties in game using VV.

Reply:
Yea, imo besides the new pda UI, the cargo ordering console looks, feels, and seems to be the best implemented, so I base most of my UI code off from it.

Reply:
Yeah, no problem here. VV's not required, but it certainly doesn't hurt.